kubernetes/k8s-check_cluster_state

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Similarly, a comment in these scripts would be nice, though somewhat less necessary.
This is "Runs cw-check_cluster_state in the first pod of any of the astaire, cassandra, or chronos services"??